src/net/peacefulcraft/cowbot/CowBot.java

```diff
@@ -111,6 +113,11 @@ public void onDisable() {
     logMessage("CowBot Disabled");
   }
 
+  // must strip message for "new" formatting characters which will slip through the legacy stripColor call. namely "&x"
+  public static String stripAmpersandBasedAndLegacyColorCodes(String message) {
```

Comment:
I put the new, shared functionality for color stripping in CowBot as a static method, but it could just be in some module both GameChatEvent and StaffChatEvent reference, if you'd prefer.